images/linux/scripts/installers/mysql.sh

```diff
@@ -4,6 +4,8 @@
 ##  Desc:  Installs MySQL Client
 ################################################################################
 
+source $HELPER_SCRIPTS/os.sh
```

Comment:
How did it work without import before?

Reply:
@maxim-lobanov 
![Screenshot 2020-12-28 at 11 34 34 AM](https://user-images.githubusercontent.com/7628945/103201387-d8240900-4900-11eb-973a-c3f8fcd14310.png)

Reply:
Nice